interface/wx/app.h

```diff
@@ -1068,6 +1068,21 @@ class wxApp : public wxAppConsole
     */
     virtual bool OSXIsGUIApplication();
 
+    /**
+        Enable the automatic tabbing features of OSX.
+
+        This feature is native to the operating system. When it is enabled, OSX
+        will automatically place windows inside tabs and show a tab bar in the
+        application. Entries are also added to the View menu to show/hide the tab bar.
+
+        @onlyfor{wxosx}
+
+        @remarks Requires OSX 10.12+.
```

Comment:
```suggestion
        @remarks Requires macOS 10.12+, does nothing under earlier OS versions.
```
I think it's not clear whether this can be called at all without the added part.